lib/OnyxCache.js

```diff
@@ -179,12 +179,23 @@ class OnyxCache {
      * Remove keys that don't fall into the range of recently used keys
      */
     removeLeastRecentlyUsedKeys() {
-        while (this.recentKeys.size > this.maxRecentKeysSize) {
-            const iterator = this.recentKeys.values();
+        let toRemove = this.recentKeys.size - this.maxRecentKeysSize;
+        if (toRemove <= 0) {
+            return;
+        }
+        const iterator = this.recentKeys.values();
+        const temp = [];
+        while (toRemove > 0) {
             const value = iterator.next().value;
             if (value !== undefined) {
```

Comment:
If `value` is undefined, should we `break`?

Reply:
it shouldn't every be undefined. I will remove it.